examples/openspending/components/DatasetsSearch.tsx

```diff
@@ -132,7 +136,10 @@ export default function DatasetsSearch({
           <div className="relative">
             <input
               aria-label="Min. date"
-              type="date"
+              type="text"
+              placeholder={minPeriod}
```

Comment:
Can we make it use the min period placeholder after cleaning the field too?

![image](https://github.com/datopian/portaljs/assets/16550934/fb6a5340-8d21-4a23-a7ee-bb659a5cb6aa)

Reply:
I tried that and it didnt work, here some of the stuff i've tried https://stackoverflow.com/questions/68480474/how-to-set-placeholder-in-input-type-date-react

Reply:
Got it. Well, it's not that bad.